### DISCUSSION TITLE
max_processes_display setting in glances.conf

### DISCUSSION THREAD

**Discussion by janreinhardt** [nicolargo/glances#3065] [General]
**Describe the bug**
 So I have configured a few things via glances.conf.
For the TASKS section, I choose 
`max_processes_display=15`
This generates a large unused space before the [alert] section „No warning … detected“. Is this a bug?

For the left column, I have a question: why is it that sometimes values are in green font colour, and some sections (DISK I/O) have values in all white colour?
And why are two of my sensor‘s values white and one green?
This situation is very uneasy to look at, can I change or configure it somehow?

**To Reproduce**
Container image 4.3.0.5 from hub.docker.com, nicolargo/glances:latest in -w mode

**Screenshots**
![IMG_2026](https://github.com/user-attachments/assets/d2793327-771c-4653-8deb-afb125798947)

**Comment by nicolargo** [comment]
Hi @janreinhardt 

the space between the processes list and the alert plugin is a "normal" behavior. The alert plugin is align to the bottom of the screen.

Only stats with thresholds defined in the Glances configuration file are display in green.  So the limit for the Disk IO should not be defined in your glances.conf. For the network, the stat for the br-55565* interface can not be retrieve so the default color (grey) is applied. 
For the sensors, there is an exception. If the threshold is not defined in the glances.conf file, the system one are applied (the one given by the sensors command line).

**Reply by janreinhardt** [reply]
> the space between the processes list and the alert plugin is a "normal" behavior. The alert plugin is align to the bottom of the screen.

Hi @nicolargo , could you please have a look at the two screenshots (MacOS Safari, 1920 x 1080 resolution):

![Screen Shot 2024-12-31 at 12 01 07 PM](https://github.com/user-attachments/assets/1bb2162c-6e83-4f33-8a01-f83729e47265)
![Screen Shot 2024-12-31 at 12 01 29 PM](https://github.com/user-attachments/assets/570398d7-a960-4057-b0e8-4a7da2300e72)

The bottom of the screen would be the end of the TASKS section. I would expect the alerts to be right there.
Instead, I have to scroll down a *huge* empty space for the alerts to show up. (= they're not at the bottom of the screen, both the left view column and the main column have ended before any scrolling is necessary)
This is normal behaviour?

Thanks!

**Reply by janreinhardt** [reply]
> For the sensors, there is an exception. If the threshold is not defined in the glances.conf file, the system one are applied (the one given by the sensors command line).

So one of my sensors (the NVME one) has got a threshold defined elsewhere? Can I delete this or re-define it to have it in white font colour just as the other sensors?
Sorry if this is a crazy question that has probably got nothing to to with Glances, but I didn't actively do any command line stuff related to that sensor. "It was just there" when I connected my PCIE SSD 🤷🏼‍♂️

**Reply by janreinhardt** [reply]
upgrading to 4.3.0.6 has solved the colour issue, all sensors are white text colour now. Nice!
